hw/top_earlgrey/ip/rv_plic/fpv/rv_plic_fpv.core

```diff
@@ -0,0 +1,28 @@
+CAPI=2:
```

Comment:
Now run `fpv_all` will directly use this top_earlgrey rv_plic fpv testbench. Only when run `fpv rv_plic_example_fpv` will use the `ip/rv_plic` parameter set.

Reply:
we could still run all fpv tests when calling `fpv_all`, even the example one (since it should still be working). WTDY?

Reply:
Yes that is correct, the fpv_all could have two versions of the rv_plic, one is the rv_plic_example, another is the rv_plic_fpv.
I do plan to add this and some other modules in the fpv_all, just waiting for my desktop to reboot lol.